src/odemis/driver/test/xt_client_test.py

```diff
@@ -712,6 +711,122 @@ def test_apply_autofocus(self):
         autofocus_state = self.microscope.is_autofocusing(channel)
         self.assertEqual(autofocus_state, False)
 
+    def test_get_mpp_orientation(self):
+        """
+        Test getting the multiprobe pattern orientation.
+        """
+        if self.xt_type != 'xttoolkit':
+            self.skipTest("This test needs XTToolkit to run.")
+
+        mpp_orientation = self.microscope.get_mpp_orientation()
+        self.assertIsInstance(mpp_orientation, float)
+        self.assertTrue(-90 < mpp_orientation < 90)
+
+    def test_mpp_orientation_info(self):
+        """
+        Test getting the multiprobe pattern orientation info.
+        """
+        if self.xt_type != 'xttoolkit':
+            self.skipTest("This test needs XTToolkit to run.")
+
+        mpp_orientation_info = self.microscope.mpp_orientation_info()
+        self.assertIsInstance(mpp_orientation_info, dict)
+        self.assertTrue("unit" in mpp_orientation_info)
+        self.assertIsInstance(mpp_orientation_info["range"], list)
+        self.assertEqual(len(mpp_orientation_info["range"]), 2)
+
+    def test_get_aperture_index(self):
+        """
+        Test getting the aperture index.
+        """
+        if self.xt_type != 'xttoolkit':
+            self.skipTest("This test needs XTToolkit to run.")
+
+        aperture_index = self.microscope.get_aperture_index()
+        self.assertIsInstance(aperture_index, int)
+        self.assertTrue(0 <= aperture_index <= 14)
+
+    def test_set_aperture_index(self):
+        """
+        Test setting the aperture index.
+        """
+        if self.xt_type != 'xttoolkit':
+            self.skipTest("This test needs XTToolkit to run.")
+
+        range = tuple(self.microscope.aperture_index_info()["range"])
```

Comment:
Because `range` is a builtin python function, I wouldn't use it as a variable name.

Reply:
Indeed, typically, we use "rng".